enforcer-rules/src/main/java/com/google/cloud/tools/dependencies/enforcer/LinkageCheckerRule.java

```diff
@@ -352,14 +339,4 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
       throw new EnforcerRuleException("Failed to collect dependency " + ex.getMessage(), ex);
     }
   }
-
-  private static ImmutableList<List<DependencyNode>> findArtifactPaths(
```

Comment:
This function has moved to DependencyGraphBuilder.
https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1210/files#diff-659d7bf4148f942fcc1eb60a50a2ee80R283